.gitignore

```diff
@@ -1,29 +1,29 @@
 # Ignore creds file
-auth/auth.yml
+*/**/auth/auth.yml
```

Comment:
I'm just curious, is this a Docker thing that it needs the extra recursion?

Reply:
Mmmm no, at least I think it doesn't. I just modified the gitignore untill it properly ignoerd sensitive data.

Reply:
Interesting. Thanks for letting me know. 👍